tests/test_widget.py

```diff
@@ -133,3 +186,53 @@ def test_calculate_length_fails_with_wrong_number_of_points(
         "ERROR: Select two points to calculate the decay length."
         in captured.out
     )
+
+
+@pytest.mark.parametrize(
+    "data_subdirs, image_count, data_loaded",
```

Comment:
```suggestion
    "data_subdirs, image_count, expect_data_loaded",
```
A bit of bike-shedding but this parameter is more like "do we expect that this succeeded" so I'd name it a bit more like that. I don't insist though.

Reply:
Had never heard the term "bike shedding" till now.